src/main/java/dk/cngroup/lentils/util/UsernameUtils.java

```diff
@@ -0,0 +1,18 @@
+package dk.cngroup.lentils.util;
+
+import java.text.Normalizer;
+
+public class UsernameUtils {
+
+    public static String generateUsername(final String oldUsername) {
```

Comment:
Šlo by na to napsat ještě pár testů?
Ještě jsem přemýšlel dát to jinam, ať to není static, ale asi je to tady OK.

Reply:
určitě, přidám testy ;)